lib/api/objectPutPart.js

```diff
@@ -201,7 +201,8 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                         // if data backend handles MPU, skip to end of waterfall
                         return next(skipError, destinationBucket,
                         partInfo.dataStoreETag);
-                    } else if (partInfo && partInfo.dataStoreType === 'azure') {
+                    } else if (partInfo &&
+                        ['azure', 'gcp'].includes(partInfo.dataStoreType)) {
```

Comment:
In the future, will we ever have to expand this list? If so, perhaps it'll be easier to configure in the future by making this a value imported from constants/config

Reply:
yes. I believe other backends  will be added in the future, so making the list an import will be better.

Reply:
thank you :D